R/net.mod.infection.R

```diff
@@ -56,7 +56,7 @@ infection.net <- function(dat, at) {
   if (nElig > 0 && nElig < nActive) {
 
     # Get discordant edgelist
-    del <- discord_edgelist(dat, at, network = 1)
+    del <- do.call(rbind, lapply(seq_along(dat$nwparam), discord_edgelist, dat = dat, at = at, include.network = TRUE))
```

Comment:
now that `dplyr` is imported by `EpiModel`, we can use `dplyr::bind_rows` instead of `do.call(rbind, list_of_dataframe)`. It makes for an easier to read code and it's way faster.

the code could be:

```r
del_lists <- lapply(seq_along(dat$nwparam), discord_edgelist, dat = dat, at = at, include.network = TRUE)
del <- dplyr::bind_rows(del_lists)
```